src/libraries/System.Formats.Cbor/tests/System.Formats.Cbor.Tests.csproj

```diff
@@ -36,18 +33,13 @@
     <Compile Include="Writer\CborWriterTests.TextString.cs" />
     <Compile Include="CoseKeyHelpers.cs" />
   </ItemGroup>
-
-  <ItemGroup Condition="'$(CborPropertyTests)' == 'True'">
-    <Compile Include="PropertyTests\CborPropertyTests.cs" />
-    <Compile Include="PropertyTests\CborRandomGenerators.cs" />
-  </ItemGroup>
-
+  
   <ItemGroup>
     <ProjectReference Include="..\src\System.Formats.Cbor.csproj" />
+    <ProjectReference Include="CborDocument\System.Formats.Cbor.Tests.DataModel.fsproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(CborPropertyTests)' == 'True'">
-    <ProjectReference Include="CborDocument\System.Formats.Cbor.Tests.DataModel.fsproj" />
+  <ItemGroup>
     <PackageReference Include="FsCheck.Xunit" Version="$(FsCheckVersion)" />
```

Comment:
Something that I missed while reviewing last time. The Version property should be `FsCheckXunitVersion` as we always use the full package name in the version property string. Just a nit.

Reply:
I used this name since `FsCheck.Xunit` is just an adapter that shares the version number with the main `FsCheck` package. I could probably make this clearer by explicitly including `FsCheck` here too.

Reply:
oh that's fine. Ignore my comment then :)